src/firefly/js/templates/lightcurve/LcResult.jsx

```diff
@@ -159,6 +159,8 @@ const buttonW = 650;
 // eslint-disable-next-line
 const StandardView = ({visToolbar, title, searchDesc, imagePlot, xyPlot, tables, settingBox}) => {
 
+    let {cutoutSize} = settingBox.props.generalEntries || '0.3';
+    //let csize = get(generalEntries, 'cutoutsize, '0.3');
```

Comment:
the commented-out code makes more sense to me.  generalEntries should be passed into StandardView so it can make use of cutoutSize.